src/controllers/hid/hidcontroller.cpp

```diff
@@ -298,10 +298,16 @@ void HidController::sendBytesReport(QByteArray data, unsigned int reportID) {
     }
 }
 
+
+namespace
+{
+    constexpr unsigned int sizeOfReportID = 1;  // Size of an USB HID Report ID is always one byte
+}
```

Comment:
- Please put this at the top of the file (below includes/forward declarations)
- I think a regular `int` suffices
- Name it `kSizeOfReportID`
- No newline before the opening brace

Reply:
Done